omniscidb/Tests/ArrowSQLRunner/ArrowSQLRunner.h

```diff
@@ -100,6 +100,7 @@ void cta(const std::string& query_string, const ExecutorDeviceType device_type);
 
 void c_arrow(
     const std::string& query_string,
+    const bool print_res,
```

Comment:
I'd avoid changing this interface. If you do want to do that, please consider making it an optional parameter with a default `false` value.

Reply:
I agree that adding a new arg for debug prints is not the best option (even if it is optional). We might want to use debug prints in many places and such args would spread all over the code. Also, having debug prints that can only be enabled through source code modifications is not really convenient. Consider using debug logging or debug config in this case. If you are not sure what would be the best option here, then simply adding a new debug function is also fine for the PR, you don't have to utilize it.